mmv1/third_party/terraform/tests/resource_workstations_workstation_test.go.erb

```diff
@@ -0,0 +1,142 @@
+<% autogen_exception -%>
+package google
+<% unless version == "ga" -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccWorkstationsWorkstation_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckWorkstationsWorkstationDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccWorkstationsWorkstation_basic(context),
+			},
+			{
+				ResourceName:            "google_workstations_workstation_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"etag"},
+			},
+      {
+				Config: testAccWorkstationsWorkstation_modified(context),
+			},
+			{
+				ResourceName:            "google_workstations_workstation_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"etag"},
+			},
+		},
+	})
+}
+
+resource "google_workstations_workstation" "default" {
```

Comment:
Looks like this chunk of config got stranded! It should be connected to `testAccWorkstationsWorkstation_modified` at the end of the file

Reply:
Oh snap, that got screwed up somehow